app/views/my_accounts/show.html.erb

```diff
@@ -2,19 +2,22 @@
 <br>
 Account balance: <%= account.balance %>
 <br>
-<% account.children.each do |child| %>
+<% accounts.each do |account| %>
```

Comment:
I have another idea - what if we just put a section below called **Shared accounts** and list these shared accounts there?

Also, later we could show not accepted invitations there as well (creating a ticket for that)